modules/flowable-bpmn-model/src/main/java/org/flowable/bpmn/model/BpmnModel.java

```diff
@@ -423,8 +423,12 @@ public void setErrors(Map<String, String> errorMap) {
     }
 
     public void addError(String errorRef, String errorCode) {
-        if (StringUtils.isNotEmpty(errorRef)) {
-            errorMap.put(errorRef, errorCode);
+        if (StringUtils.isNotEmpty(errorRef) && !errorMap.containsKey(errorRef)) {
```

Comment:
What I am not sure about is whether we can merge errors in this way.

Reply:
@martin-grofcik  I think it is correct